README.md

```diff
@@ -105,6 +105,17 @@ it('should demonstrate this matcher`s usage with react testing library', async (
 > Note: If you're using `react testing library` <9.0.0 you should be using the
 > [`cleanup`](https://testing-library.com/docs/react-testing-library/api#cleanup) method. This method removes the rendered application from the DOM and ensures a clean HTML Document for further testing.
 
+In case you're using this library to verify some component that relies on [React Portals](https://reactjs.org/docs/portals.html#gatsby-focus-wrapper), make sure you supply [`baseElement`](https://testing-library.com/docs/react-testing-library/api#baseelement) instead of `container`:

```

Comment:
```suggestion
If you're using [React Portals](https://reactjs.org/docs/portals.html), use the [`baseElement`](https://testing-library.com/docs/react-testing-library/api#baseelement) instead of `container`:

```